dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/TrnToken/CheckAnswers.cshtml

```diff
@@ -41,6 +41,13 @@
                         <govuk-summary-list-row-action href="" visually-hidden-text="last names">Change</govuk-summary-list-row-action>
                     </govuk-summary-list-row-actions>
                 </govuk-summary-list-row>
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Preferred name</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>@Model.PreferredName</govuk-summary-list-row-value>
```

Comment:
Does this have a fallback if not set? If not, can we have `Not provided` in grey text?

Reply:
I think technically this should always be set when you reach the trntoken/check-answers page (same for register/check-answers). Can add a fallback for both though.

Regarding grey text, we have fallbacks for e.g. National Insurance number but text colour is black